apps/docs/next.config.js

```diff
@@ -85,6 +85,22 @@ const nextConfig = {
 				destination: '/reference/:slug*',
 				permanent: true,
 			},
+			// redirects at the top level
+			{
+				source: '/examples',
+				destination: '/examples/basic/basic',
```

Comment:
this should redirect to `/examples/` not basic/basic - since it's a permanent redirect we should redirect to something that'll be more evergreen